perception/object_segmentation/elevation_map_loader/launch/elevation_map_loader.launch.xml

```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="pointcloud_map_path" description="point cloud map file or directory path" />
```

Comment:
This arg is no more needed. Please remove it.

Reply:
please refer to https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/7160996b9bb6b72a5df9362b23bbd2bccdd3c792